src/sqon/searchSqon.ts

```diff
@@ -49,7 +49,7 @@ export const searchSqon = async (
     return ids;
 };
 
-const runQuery = ({ query, variables, mock, project }) => {
+export const runQuery = ({ query, variables, mock, project }) => {
```

Comment:
this function should be async I think or we could remove all await we put when calling it